torchci/pages/api/track.ts

```diff
@@ -0,0 +1,19 @@
+import getRocksetClient from "lib/rockset";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const body = JSON.parse(req.body);
+  const timestamp = Date.now();
+
+  if (req.method == "POST" && body.url.startsWith("https://hud.pytorch.org")) {
```

Comment:
Let's put a TODO here if you don't want to do this as part of this PR, but I think we should eventually write this to `hud_tracking` DynamoDB table and sync that with Rockset instead.  https://github.com/pytorch/test-infra/blob/main/torchci/lib/dynamo.ts can just be used after creating the table (preferably using Terraform)